_zip/missingapi.yml

```diff
@@ -2,44 +2,44 @@
 hrefUpdated: true
 references:
 - uid: System.Diagnostics.PresentationTraceSources.TraceLevel*
-  href: https://msdn.microsoft.com/library/system.diagnostics.presentationtracesources.tracelevel.aspx
+  href: /previous-versions/bb778381(v=vs.110)
```

Comment:
This file isn't published either, so probably needs to be reverted.

Reply:
It is there, just that it's a relative link (and cross-repo). Once in master (and live) it will resolve correctly.